src/core/credentials/transport/tls/ssl_utils.h

```diff
@@ -100,6 +102,7 @@ grpc_security_status grpc_ssl_tsi_server_handshaker_factory_init(
     tsi::TlsSessionKeyLoggerCache::TlsSessionKeyLogger* tls_session_key_logger,
     const char* crl_directory, bool send_client_ca_list,
     std::shared_ptr<grpc_core::experimental::CrlProvider> crl_provider,
+    std::shared_ptr<grpc_core::SpiffeBundleMap> spiffe_bundle_map,
```

Comment:
We should combine the `pem_root_certs` and `spiffe_bundle_map` arguments using the same `std::shared_ptr<RootCertInfo>` type I suggested elsewhere.